Library/Formula/docker-machine-driver-xhyve.rb

```diff
@@ -0,0 +1,38 @@
+require "language/go"
+
+class DockerMachineDriverXhyve < Formula
+  desc "Docker Machine driver for xhyve"
+  homepage "https://github.com/zchee/docker-machine-driver-xhyve"
+  url "https://github.com/zchee/docker-machine-driver-xhyve/releases/download/v0.2.1/docker-machine-driver-xhyve"
+  sha256 "d56db4635ad2f711d73b44508405277bddd5af439af2c82dbdd0da116e8d1380"
+  head "https://github.com/zchee/docker-machine-driver-xhyve.git"
+
+  depends_on "docker-machine"
+  depends_on "go" => :head
```

Comment:
What are you trying to do here? It needs `go` for the `head` compile or it needs a `head` build of `go` to install?